src/__tests__/posthog-core.js

```diff
@@ -277,6 +277,41 @@ describe('capture()', () => {
         const event = given.subject()
         expect(event.properties.key.length).toBe(50000)
     })
+
+    describe('url_allowlist matching event names', () => {
+        beforeEach(() => {
+            given.lib.__compress_and_send_json_request = jest.fn()
+            given('config', () => ({
+                property_blacklist: [],
+                event_capture_config: { url_allowlists: { $pageview: ['https://posthog.com/test/*'] } },
```

Comment:
This will also match `http://posthog.com/testnotwhatiwanted` and `http://posthog.com/test//////////`